physics/module_mp_nssl_2mom.F90

```diff
@@ -8118,7 +8118,9 @@ subroutine radardd02(nx,ny,nz,nor,na,an,temk,         &
 !          write(0,*) 'Hail,snow c: ',an(ix,jy,kz,lnh),an(ix,jy,kz,lns)
 !          write(0,*) 'dtmps,dtmph = ',dtmps,dtmph
 !         ENDIF
-        IF ( .not. dtmp(ix,kz) .lt. 1.e30 .or. dbz(ix,jy,kz) > 190.0 ) THEN
+
+! 1.2 GB of messages is a bit too much.
```

Comment:
Ha. Agreed. Why can't a debug flag be used to control this output? Also, probably don't need to keep the comment in the repo.

Reply:
Yes, I can use `ndebug`. I'm compiling that now to make sure it works.

Reply:
We temporally added these when we debugged the MYNN coupling to NSSL MP in PR #43. I will restore it in this PR.

Reply:
@haiqinli - don't restore the line. I'm adding `ndebug>2` to it, and I'll merge that change in a few minutes.

Reply:
The gnu all suites build passed, so I pushed this change.